.docker/Dockerfile.local

```diff
@@ -1,20 +1,55 @@
-FROM node:8
+FROM ubuntu:16.04 as builder
```

Comment:
*NO PR*

We're using our own docker configuration to build the chat completely in the builder.
RC itself just downloads the source from its releases in order to create the docker image.